lib/irb/cmd/exit_forced_action.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require_relative "nop"
+
+module IRB
+  # :stopdoc:
+
+  module ExtendCommand
+    class ExitForcedAction < Nop
+      category "IRB"
+      description "Exit the current irb session with exit!."
```

Comment:
I think we can make the effect of this command more clear:

```suggestion
      description "Exit the current process."
```